doc/classes/FileDialog.xml

```diff
@@ -133,6 +136,15 @@
 		<constant name="ACCESS_FILESYSTEM" value="2" enum="Access">
 			The dialog allows accessing files on the whole file system.
 		</constant>
+		<constant name="CASE_SENSITIVITY_TRUE" value="0" enum="CaseSensitivity">
+			The dialog will use a case sensitive check against the filters when saving.
+		</constant>
+		<constant name="CASE_SENSITIVITY_FALSE" value="1" enum="CaseSensitivity">
+			The dialog will use not a case sensitive check against the filters when saving.
+		</constant>
```

Comment:
Generally speaking these are pretty weird names for constants. It's better to be explicit, e.g. `CASE_SENSITIVITY_SENSITIVE`/`CASE_SENSITIVITY_INSENSITIVE`.